setupJestMock.ts

```diff
@@ -0,0 +1,17 @@
+const localStorageMock = {
```

Comment:
Since the initial state calls local storage, and the utils file needs a dispatcher, this sadly forces Jest to call local storage which does not exist. This mock is put in place to fix it.